src/test/spec/client_side_encryption.rs

```diff
@@ -28,3 +33,29 @@ fn spec_predicate(test: &TestCase) -> bool {
     }
     true
 }
+
+#[cfg_attr(feature = "tokio-runtime", tokio::test(flavor = "multi_thread"))]
+#[cfg_attr(feature = "async-std-runtime", async_std::test)]
+async fn run_legacy() {
+    let _guard: RwLockWriteGuard<()> = LOCK.run_exclusively().await;
+
+    run_spec_test_with_path(&["client-side-encryption", "legacy"], |path, test| async {
+        if path.ends_with("timeoutMS.json") {
```

Comment:
can we add a TODO for RUST-582 to unskip this?

Reply:
Done.